recipe/meta.yaml

```diff
@@ -29,6 +29,10 @@ requirements:
     - python-confluent-kafka  # [not win]
     - redis-py
     - slack-sdk
+    - globus-sdk
```

Comment:
We need to build it based on https://github.com/conda-forge/globus-sdk-feedstock/blob/master/recipe/meta.yaml.